src/Sylius/Bundle/ThemeBundle/composer.json

```diff
@@ -36,14 +36,16 @@
         "phpunit/phpunit": "^4.1",
         "symfony/twig-bundle": "^2.7.7"
     },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
```

Comment:
It's definitely a good idea, without prefer-stable we will install unstable versions of all our dependencies, whereas stable versions suits as well. We should install dev versions only if there isn't any other version available.